ia-guide/ia-static.md

```diff
@@ -0,0 +1,341 @@
+
+Put these things right up front in your docs, so your readers can go away and learn something else first if they need to.
+And be prepared to be challenged: you might discover in time that your assumptions are incorrect, and if that happens, you'll need to revisit your docs, too.
+
+## Who are you writing for?
+
+Now that you know what you're writing about, consider who you're writing for.
+Remember that people come in all shapes and sizes, so let's think up three different people, to try and make sure we're covering all the options.
+
+![A winking smiling person wearing a purple shirt and pants, with pastel coloured hair, waving](images/wave.svg "Wave")
+
+The way you categorize your three readers is up to you, but sometimes the best way to start is by thinking of a beginner, an intermediate reader, and an expert.
+Or, depending on the kind of product you have, you might think of an ordinary user, a system administrator, and a support person.
+
+Your three readers should probably be interested in different parts of your product.
+They will be trying to do different tasks, and they will need different levels of help from your docs.
+Don't be afraid to let them overlap a little bit, but try and make sure you've covered most of the options.
+
+Let's take a closer look at our readers, and work out why they read the docs.
+
+I'm going to tell you a secret:
+
+*No one curls up at night with a warm drink and a great technical manual.*
+
+If that's the case, why **do** people read docs?
+Generally speaking, it's because they want to achieve something.
+
+To put that in less abstract terms, you need to work out what problem your reader is trying to solve.
+If Dusty wants to put their degree up on the wall, they might want to consult some documentation about how to do it.
+The documentation Dusty needs to do this would probably not be called "How to Choose a Drill".
+The documentation Dusty needs is "How to Hang a Picture".
+
+This is an important distinction, because if you get it wrong, your readers won't know if the content is right for them or, worse, they won't find your content at all.
+
+![A brown rat with white wings and a colorful unicorn horn](images/unirat.svg "Unirat")
+
+So let's think about what our readers are trying to achieve.
+
+For each of your readers, start by thinking about how they might begin.
+What is the very first thing they might want to do?
+Let's assume that they manage to complete that without any trouble.
+What would be the second thing that they would want to do?
+And now that they've managed to complete that what is the next thing?
+
+When you've got tasks for all of your readers, let's put it all together.
+
+## Writing for your readers
+
+Now that you know who your readers are, and the kinds of things they might be using your documentation for, you can display them in a table like this:
```

Comment:
One thought to add to the table might be a doc-type to correspond with some of the templates we are promoting from the good-docs/templates (example from `Readme`)

| Template name | Documentation type | Description |
| ------------- | ------------------ | ----------- |
| API Project overview | Concept | An overview of your API |
| API Quickstart | Concept, Task | Simplest possible method of implementing your API |
| API Reference | Reference | List of references related to your API |
| Explanation | Concept | Longer document giving background or context to a topic |
| How-to | Task | Short series of steps for a particular task |
| Tutorial | Concept, Task | A training document for a product or topic |
| General reference entry | Reference | Specific details about a particular topic |
| Logging reference | Reference | Description of log pipelines |

Not say we should support this specifically, but the idea that a document type can help indicate the object.